ios/RNCConnectionState.m

```diff
@@ -62,9 +62,13 @@ - (instancetype)initWithReachabilityFlags:(SCNetworkReachabilityFlags)flags
                     _cellularGeneration = RNCCellularGeneration3g;
                 } else if ([netinfo.currentRadioAccessTechnology isEqualToString:CTRadioAccessTechnologyLTE]) {
                     _cellularGeneration = RNCCellularGeneration4g;
-                } else if ([netinfo.currentRadioAccessTechnology isEqualToString:CTRadioAccessTechnologyNRNSA] ||
-                           [netinfo.currentRadioAccessTechnology isEqualToString:CTRadioAccessTechnologyNR]) {
-                    _cellularGeneration = RNCCellularGeneration5g;
+                } else if (@available(iOS 14.1, *)) {
+                    if ([netinfo.currentRadioAccessTechnology isEqualToString:CTRadioAccessTechnologyNRNSA] ||
+                        [netinfo.currentRadioAccessTechnology isEqualToString:CTRadioAccessTechnologyNR]) {
+                        _cellularGeneration = RNCCellularGeneration5g;
+                    }
+                } else {
```

Comment:
Not necessary?

Reply:
oh good point - I took the "Xcode automatic refactoring" suggestion verbatim and did not realize it put in the else branch for me. You are right and I will remove the else branch